### ISSUE TITLE
Handler for when webview is closed

### ISSUE THREAD

**Issue by cameronmcefee** [skpm/sketch-module-web-view#3]
Hello! I've been poking around trying to find a solution for this, but haven't figured it out. Is it possible with the current set up to register a handler that runs whenever the webview is closed with `esc` or the close button?

**Comment by mathieudutour** [comment]
Hum I've never done it. But it should be possible with a [NSWindowDelegate](https://developer.apple.com/reference/appkit/nswindowdelegate) on the panel

**Comment by cameronmcefee** [comment]
:cool: Will do. Thanks!

**Comment by bomberstudios** [comment]
Just came here to say I'd also like to see this implemented : )

**Comment by timohofmeijer** [comment]
Could not get it to work via WindowDelegate. I hacked it in by passing a `closeHandler` `option` to the `WebUI` creator function. It gets trigger by `cmd-w` as well.
```
if (options.closeHandler) {
    const closeButton = panel.standardWindowButton(NSWindowCloseButton)
    closeButton.setCOSJSTargetFunction(function(sender) {
      options.closeHandler()
      COScript.currentCOScript().setShouldKeepAround(false)
      threadDictionary.removeObjectForKey(options.identifier)
      panel.close()
    })
    closeButton.setAction("callAction:")
}
```

**Comment by mathieudutour** [comment]
oh nice! Do you want to open a PR?

**Comment by mathieudutour** [comment]
fixed on v0.1.6 :tada: thanks @timohofmeijer!
